test/integration/index.spec.js

```diff
@@ -3268,6 +3335,31 @@ describe("Test file uploading", () => {
 			});
 	});
 
+	it("should send data field with multipart", () => {
+		return request(server)
+			.post("/upload")
+			.attach("myFile", assetsDir + "logo.png")
+			.field("name", "moleculer")
+			.then(res => {
+				expect(res.statusCode).toBe(200);
+				expect(res.headers["content-type"]).toBe("application/json; charset=utf-8");
+				expect(res.body).toEqual('moleculer');
```

Comment:
should check that file is uploaded successfully, as well.

Reply:
Done!